WordPress/gutenberg
# Show the content of the paragraph and list items in the list view.

## What?


I've recently found myself using the blocks list a lot. I find it super helpful. But I ended up experiencing that it was challenging to identify essential elements like paragraphs and list items for more complex layouts. I started looking around and found some conversations about [having custom naming/labeling](https://github.com/WordPress/gutenberg/issues/33583). That would probably be the ultimate solution for group blocks or other blocks. 

For text content-oriented blocks, more precisely the paragraph block and the list items, I feel that the default solution would be using the actual content in the list view. And this PR addresses that change.

## Why?


The paragraph and list-item (`<li>`) blocks will be better represented in the list view. It should enhance its functionality by adding an instant visual representation of the block content.

## Testing Instructions





1. Create a post or page.
2. Open the list view.
3. Add a new paragraph or a list and see how the changes in the content are reflected in the list view.

## Screenshots or screencast 

https://user-images.githubusercontent.com/252415/234868714-e03f7de4-af01-40df-af1c-28ce84f7b472.mov
